src/webgpu/web_platform/reftests/canvas_display_after_device_lost.html.ts

```diff
@@ -10,6 +10,7 @@ void (async () => {
 
   const adapter = await navigator.gpu.requestAdapter();
   assert(adapter !== null);
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
Should we just add a .eslintrc.js to the reftests folder that turns off this rule?

Reply:
Oh yes, that would be a good idea. I sort of considered it but forgot (I'd have to turn off the entire no-restricted-syntax rule but that seems fine.)